examples/ESP8266MeshHelloWorld/README.md

```diff
@@ -2,11 +2,35 @@
 
 This is a trivial node that can be used to test the mesh network.
 
-## Configuration
-a sample configuration is provided in `src/credentials.h.example`.
+![](http://i.imgur.com/w0uKBjU.png)
+
+If nodes can see the router, they communicate directly (Ex: 1 ->2). If not, they pass messages via the mesh network to other nodes. (Ex: 1 -> 2 -> 3)
+
+## QUICKSTART
+This describes one fast and easy way to get started, though there are other options.
+
+You will require:
+- A wirless router, with a dedicated secured (WPA2, AES) 2.4Ghz network, set to control channel 1 (2.412 Ghz), auto 20/40Mhz mode
+- A computer (Linux or Windows tested) connected to the above network
+- 1 or more ESP8266 chips to test with
 
-1) First copy `src/credentials.h.example` to `src/credentials.h`
-2) Edit `credentials.h` and modify the configuration to suit your environment
+1) Install the IDE: PlatformIO on the computer http://platformio.org/get-started/ide?install=atom
+1b) Install the ESP8266MQTTMesh Library. PlatfromIO Home -> Libraries -> Search -> ```ESP8266MQTTMesh``` -> Install\
+This should automatically install all necessary pre-requisites. 
+2) Install the MQTT Broker: Mosquitto and all pre-requisites https://mosquitto.org/download/
+3) Install the MQTT UI/Monitor: MQTT.fx http://www.mqttfx.org/
```

Comment:
I think that installing MQTT is beyond the scope of what I'd expect in a tutorial...
MQTT.fx is certainly not necessary